compiler/src/iree/compiler/InputConversion/StableHLO/StableHLOToLinalgReduce.cpp

```diff
@@ -85,6 +103,8 @@ struct ReduceOpToGenericConverter final
   LogicalResult matchAndRewrite(
       mlir::stablehlo::ReduceOp op, OpAdaptor adaptor,
       ConversionPatternRewriter &rewriter) const override {
+    if (isUnsupported(op)) return failure();
```

Comment:
probably worth rewriter.notifyMatchFailure on these?

Reply:
Good point. I'd expect this to be exceedingly rare, so we should make the failure prominent. Done.